assets/js/components/BatchEdit/About/CoreMetadata.jsx

```diff
@@ -45,6 +52,25 @@ const BatchEditAboutCoreMetadata = ({ ...restProps }) => {
           label="Alternate Title"
         />
       </div>
+      <div className="column is-full">
+        <UIFormField label="Collection">
+          <UIFormSelect
+            isReactHookForm
+            name="collectionId"
```

Comment:
Semantically a form `name` should just describe the data.  We don't need the "Id".  So let's make `name="collection"` instead.